### DISCUSSION TITLE
tab-give hotkey with id = 0

### DISCUSSION THREAD

**Discussion by joaonunatings** [qutebrowser/qutebrowser#7475] [Q&A] [answered]
Hi, I'm starting to use the 'tab-give' using the hotkey (_gD_)
The only problem I encounter was when trying to 'give a tab' to a window with id = 0.
When using the hotkey you first enter the window id then press gD. However 0 is not registered, any tips?

**Comment by The-Compiler** [comment, marked as answer]
Counts are offset by one, so a count of 1 will actually select window 0. See also:

- #4529
